lib/chat/client/chat-ui.js

```diff
@@ -314,7 +315,9 @@ class ChatUi
     useGlobalRoomForMessage(message)
     {
         // if is global check the global chat room:
-        let globalChat = sc.get(this.gameManager.joinedRooms, ChatConst.CHAT_GLOBAL, false);
+        // TODO: See how we can get the joinedRooms from here or from uiSceneManager;
```

Comment:
let's discuss it, passing the whole gameManager's object could lead to plugins modifying some properties of it. Shouldn't we prevent those behaivours?

Reply:
nop, that's the entire idea, everything is referenced, so anything will impact on the game behavior, you can modify the entire game client at any point, even if you try to block users from it, there's no way to avoid it :P

Reply:
Done.